### PR TITLE
Equalizer added

### PR DESCRIPTION
**Pull request by renzowolox**
Equalizer agregado

### CODE REVIEW

File: maquetado/scss/styles.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.title {
+  color: red;
+}
```
Comment (Dorian30) [inline maquetado/scss/styles.scss]: Be careful with this 👀Missing white line here.

Comment (renzowolox) [reply maquetado/scss/styles.scss]: Ok, I'll be careful with this.

File: maquetado/scss/application.scss [hunk 2]
```diff
@@ -1,3 +1,2 @@
-.title {
-  color: red;
-}
+@import 'equalizer';
+@import 'styles';
```
Comment (leonardofrp5) [inline maquetado/scss/application.scss]: you can leave a space at the end

File: maquetado/scss/equalizer.scss [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+li {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+::placeholder {
+  color: $equalizer-grey;
+  opacity: 1;
+}
```
Comment (leonardofrp5) [inline maquetado/scss/equalizer.scss]: same 👆

File: maquetado/scss/styles.scss (hunk 1 above)
Comment (leonardofrp5) [inline maquetado/scss/styles.scss]: same 👆

**Pull request merged**